plugins/theme-hackermode/src/index.js

```diff
@@ -0,0 +1,118 @@
+private:true 
```

Comment:
This needs to be added to the `package.json` file as a config property!

E.g.:

```json
  "main": "./dist/index.js",
  "module": "./src/index.js",
  "unpkg": "./dist/index.js",
  // This added.
  "private": true,
  "author": "",
```